src/pages/Exercise3.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, useState } from 'react';
+import '../styles/Exercise3.scss';
+import Pipi from '../../public/Pipi.svg';
+import AppWrapper from '../components/AppWrapper';
+import Box from '../components/Box';
+import Grid from '../components/Grid';
+import HintBox from '../components/HintBox';
+import NavButtons from '../components/NavButtons';
+import RunCode from '../components/RunCode';
+import { HeaderSections } from '../types/globalTypes';
+
+const question = [
+  {
+    options: [
+      '*basketballPtr = soccerBall;',
+      'basketballPtr = soccerBall;',
+      'basketballPtr = &soccerBall;',
+      '&basketballPtr = soccerBall;',
+    ],
+    answer: '*basketballPtr = soccerBall;',
+    correctText:
+      'Correct! This correctly dereferences the pointer and sets the basketball to a soccer ball.',
+    wrongText: 'Not quite',
```

Comment:
Will need to change later but don't worry about this for now.

Reply:
got it! thanks a lot for reviewing my code!